pkgs/by-name/et/etterna/package.nix

```
@@ -0,0 +1,111 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  makeDesktopItem,
+  copyDesktopItems,
+  # deps
+  clang,
+  cmake,
+  python2,
```

```suggestion
  cmake,
```
Unused arguments. LGTM otherwise :)